### DISCUSSION TITLE
Simpler alternative to Multibuffers

### DISCUSSION THREAD

**Discussion by AkisArou** [zed-industries/zed#31514] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

Hello, you are doing amazing work on this editor!

I know **multibuffers** are useful for many people, but I think it would be nice to provide a simpler alternative (VSCode-like) for example for Find References etc!
This should help for faster navigation between references with less overhead.
Thanks!

**Comment by JosephTLyons** [comment]
Hey @AkisArou, with Zed's current multi-buffer, I'm curious to hear how do you end up navigating to a symbol found in a `Find All References` action.

**Comment by AkisArou** [comment]
Hey!
I use vim motions, so I use `ctrl-d` and `ctrl-u` to scroll up and down.

**Comment by marcospb19** [comment]
> This should help for faster navigation between references with less overhead.

We do have faster navigation between references if you happen to open the outline panel when you have the multibuffer.

The current problems with the outline panel, IMO, are:

1. It's impossible to discover this on your own.
    - We can add a better banner that explains how it works.
2. Takes space that you might want for another panel.
3. It's hard for newcomers to find the hotkeys to toggle it.

Before creating something new, we could look at these problems and see if solving suffices.

((I think some people would prefer having the outline panel expand automatically, I talked about this internally, but folks are reluctant as it is potentially annoying and distracting.))

**Comment by probably-neb** [comment]
Hey, I'm going to move this to a discussion. Feel free to keep discussing what this could look like and why it is beneficial
